tests/acceptance/bootstrap/OcmContext.php

```diff
@@ -230,6 +230,25 @@ public function userFindsAcceptedUsers(string $user): void {
 		$this->featureContext->setResponse($this->findAcceptedUsers($user));
 	}
 
+	/**
+	 *
+	 * @param string $user
+	 * @param string $ocmUserName
+	 *
+	 * @return array
+	 * @throws GuzzleException
+	 */
+	public function getAcceptedUserByName(string $user, $ocmUserName): array {
+		$this->userFindsAcceptedUsers($user);
+		$responseArray = ($this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse()));
```

Comment:
It's a helper function so it's better to not save the response here
```suggestion
		$response = $this->findAcceptedUsers($user);
		$responseArray = ($this->featureContext->getJsonDecodedResponse($response));
```